Add Flux to chart gitops docs (#24288)
mistercrunch/Airflow: docs/helm-chart/index.rst

The command removes all the Kubernetes components associated with the chart and deletes the release.

Installing the Chart with ArgoCD or FluxCD

> --------------------------------

Reviewer: ```suggestion
------------------------------------------
```

> Installing the Chart with ArgoCD or FluxCD
> ------------------------------------------

Reviewer 2: ```suggestion
Installing the Chart with Argo CD or Flux
-----------------------------------------
```

Just from looking at their site, they seem to just use "Flux", not "FluxCD"?
Also looks like it should be "Argo CD" instead too.

Author: Agreed. They used to use FluxCD in the beginning.

> When installing the chart using ArgoCD or FluxCD, you MUST set the two following values, or your application

Reviewer 2: ```suggestion
When installing the chart using Argo CD or Flux, you MUST set the two following values, or your application
```
